commit b79375827d931dba8eee3ff4e38b243e24d26b5c
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Tue Oct 5 09:31:19 2010 -0700

    Staging: brcm80211: remove unneeded #ifdef checks
    
    For a .h file, you never need to check it, the .h file does it
    itself.  For the typedef.h file, this is never needed.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/msgtrace.h
+++ b/drivers/staging/brcm80211/include/msgtrace.h
@@ -17,9 +17,7 @@
 #ifndef	_MSGTRACE_H
 #define	_MSGTRACE_H
 
-#ifndef _TYPEDEFS_H_
 #include <typedefs.h>
-#endif
 
 /* This marks the start of a packed structure section. */
 #include <packed_section_start.h>
--- a/drivers/staging/brcm80211/include/proto/802.11.h
+++ b/drivers/staging/brcm80211/include/proto/802.11.h
@@ -17,12 +17,8 @@
 #ifndef _802_11_H_
 #define _802_11_H_
 
-#ifndef _TYPEDEFS_H_
 #include <typedefs.h>
-#endif
-
 #include <proto/wpa.h>
-
 #include <packed_section_start.h>
 
 #define DOT11_A3_HDR_LEN		24
--- a/drivers/staging/brcm80211/include/proto/bcmeth.h
+++ b/drivers/staging/brcm80211/include/proto/bcmeth.h
@@ -17,10 +17,7 @@
 #ifndef _BCMETH_H_
 #define _BCMETH_H_
 
-#ifndef _TYPEDEFS_H_
 #include <typedefs.h>
-#endif
-
 #include <packed_section_start.h>
 
 #define	BCMILCP_SUBTYPE_RATE		1
--- a/drivers/staging/brcm80211/include/proto/bcmevent.h
+++ b/drivers/staging/brcm80211/include/proto/bcmevent.h
@@ -17,10 +17,7 @@
 #ifndef _BCMEVENT_H_
 #define _BCMEVENT_H_
 
-#ifndef _TYPEDEFS_H_
 #include <typedefs.h>
-#endif
-
 #include <packed_section_start.h>
 
 #define BCM_EVENT_MSG_VERSION		1
--- a/drivers/staging/brcm80211/include/proto/bcmip.h
+++ b/drivers/staging/brcm80211/include/proto/bcmip.h
@@ -17,10 +17,7 @@
 #ifndef _bcmip_h_
 #define _bcmip_h_
 
-#ifndef _TYPEDEFS_H_
 #include <typedefs.h>
-#endif
-
 #include <packed_section_start.h>
 
 #define IP_VER_OFFSET		0x0
--- a/drivers/staging/brcm80211/include/proto/ethernet.h
+++ b/drivers/staging/brcm80211/include/proto/ethernet.h
@@ -17,9 +17,7 @@
 #ifndef _NET_ETHERNET_H_
 #define _NET_ETHERNET_H_
 
-#ifndef _TYPEDEFS_H_
 #include "typedefs.h"
-#endif
 #include <packed_section_start.h>
 
 #define	ETHER_ADDR_LEN		6
--- a/drivers/staging/brcm80211/include/proto/vlan.h
+++ b/drivers/staging/brcm80211/include/proto/vlan.h
@@ -17,10 +17,7 @@
 #ifndef _vlan_h_
 #define _vlan_h_
 
-#ifndef _TYPEDEFS_H_
 #include <typedefs.h>
-#endif
-
 #include <packed_section_start.h>
 
 #define VLAN_VID_MASK		0xfff
